src/core/components/courses/ProfileTraineeFollowUp.vue

```diff
@@ -26,7 +26,8 @@
           @click="attendanceSheetAdditionModal = true" />
       </div>
     </div>
-    <trainee-follow-up-table :learners="learners" :loading="loading" class="q-my-md" />
+    <trainee-follow-up-table :learners="learners" :loading="loading" class="q-my-md"
+      :is-blended="true" />
```

Comment:
Il me semble que quand c'est un boolean tu n'as pas besoin de mettre `:is-blended="true"` tu peux mettre directement `is-blended`